airbyte-webapp/src/components/connectorBuilder/Builder/StreamSlicerSection.tsx

```diff
@@ -113,8 +118,7 @@ export const StreamSlicerSection: React.FC<StreamSlicerSectionProps> = ({ stream
             type="string"
             path={buildPath("step")}
             label="Step"
-            tooltip="Time interval for which to break up stream into slices, e.g. 1d"
-            pattern={timeDeltaRegex}
+            tooltip="Time interval for which to break up stream into slices, e.g. P1D for daily slices"
```

Comment:
nit: add the name of the format to the description
```suggestion
            tooltip="Time interval (ISO 8601 duration) for which to break up stream into slices, e.g. P1D for daily slices"
```